assets/js/apps/tiles/components/DatasetFilter.jsx

```diff
@@ -18,81 +18,70 @@ const DatasetFilter = ({ onFilter, onSort }) => {
   };
   return (
     <Fragment>
-      <p>
-        <div className="row">
-          <div className="form-horizontal col-md-8">
-            <div className="form-group has-search">
-              {/* <span className="fa fa-search form-control-feedback"/> */}
-              <input
-                type="text"
-                className="form-control"
-                placeholder="Just start typing to filter datasets based on description"
-                onKeyUp={onFilter}
-              />
-            </div>
+      <div className="row mb-3">
+        <div className="col-md-8">
+          <div className="has-search">
+            <input
+              type="text"
+              className="form-control"
+              placeholder="Start typing to filter datasets based on description"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 28 spaces but found 14.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
                            placeholder="Start typing to filter datasets based on description"
```